model/src/main/kotlin/config/ScannerConfiguration.kt

```diff
@@ -65,6 +66,7 @@ data class ScannerConfiguration(
      * A list of glob expressions that match file paths which are to be excluded from scan results.
      */
     val ignorePatterns: List<String> = listOf(
+        "**/*.ort.yml",
```

Comment:
Does that actually also match a `.ort.yml` file in the root?

Reply:
Yes it does.